core-bundle/src/Resources/contao/library/Contao/Pagination.php

```diff
@@ -386,6 +422,11 @@ public function getItemsAsArray()
 		return $arrLinks;
 	}
 
+	public function getContextForTwigComponent(Request $request): ?array
```

Comment:
What's this? I mean, why does it have a parameter that's not used? Why can't it just call `->compile()` directly?

Reply:
Yep, you are right, this is a left-over. Removed in [8653532](https://github.com/contao/contao/pull/4847/commits/8653532e703255ee582a929c9123a49b83caf521)